src/http_parser.cpp

```diff
@@ -130,6 +130,23 @@ namespace libtorrent {
 		return url;
 	}
 
+	std::string const& http_parser::header(const string_view key) const
+	{
+		static std::string empty;
+		auto const i = m_header.find(key.to_string());
```

Comment:
would you mind leaving a TODO comment here that with C++14 the string construction should be removed?

i.e. when using a transparent (``is_transparent``) comparator, a ``string_view`` can be used directly here